distros/spring/pom.xml

```diff
@@ -54,9 +54,20 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
-            <version>2.1.6.RELEASE</version>
+            <version>2.1.7.RELEASE</version>
```

Comment:
Is there a reason for not using the `spring-boot.version` variable here?

Reply:
This dependency is used for test and doesn't explicitly have to be tied to the version defined in properties. I assumed that was the reasoning that led to the property not being used before I arrived in the scene. We can change it, but given the test scope I feel it doesn't have to be set to the parent root version.